src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/audit/Audit.kt

```diff
@@ -1,4 +1,4 @@
 package uk.gov.justice.digital.hmpps.nomisprisonerapi.audit
 
-@Target(allowedTargets = [AnnotationTarget.FUNCTION])
+@Target(allowedTargets = arrayOf(AnnotationTarget.FUNCTION))
 annotation class Audit()
```

Comment:
a desperate attempt to stop IntelliJ complaining about this annotation that compiles perfectly fine